easybuild/easyconfigs/g/GD/GD-2.52-foss-2016b-Perl-5.24.0.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'PerlModule'
+
+name = 'GD'
+version = '2.52'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://search.cpan.org/~lds/%(name)s/'
+description = """GD.pm - Interface to Gd Graphics Library"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['http://cpan.metacpan.org/authors/id/L/LD/LDS/']
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['646a4b5ba587763ed791bad4f2f79e95']
+
+dependencies = [
+    ('Perl', '5.24.0'),
+    ('libgd', '2.2.3'),
```

Comment:
I get an error during the sanity check of libgd 2.2.4 (the log file contains no further information.):

```
== FAILED: Installation ended unsuccessfully (build directory: /dev/shm/libgd/2.2.4/foss-2016b): build failed (first 300 chars): Sanity check failed: no file of ('lib/libgd.%(SHLIB_EXT)s',) in /easybuild/test/software/libgd/2.2.4-foss-2016b
```

But the file is actually there:

```
$ ls /easybuild/test/software/libgd/2.2.4-foss-2016b/lib/
libgd.a  libgd.la  libgd.so  libgd.so.3  libgd.so.3.0.4  pkgconfig
```

Shouldn't `SHLIB_EXT` resolve to `so`? The sanity check is this:

```
sanity_check_paths = {
    'files': ['lib/libgd.a', 'lib/libgd.%(SHLIB_EXT)s'],
    'dirs': ['bin', 'include'],
}
```